lib/workflows/hv_patch_and_reboot.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+
+from icinga_api.downtime import schedule_downtime, remove_downtime
+from structs.icinga.downtime_details import DowntimeDetails
+from structs.icinga.icinga_account import IcingaAccount
+from structs.ssh.ssh_connection_details import SSHDetails
+from ssh_api.exec_command import SSHConnection
+
+
+def patch_and_reboot(
+    icinga_account: IcingaAccount,
+    name: str,
+    host: str,
+    private_key_path: str,
+):
+    connection_details = SSHDetails(
+        host=host, username="stackstorm", private_key_path=private_key_path
+    )
+    ssh_client = SSHConnection(connection_details)
+    start_time = datetime.datetime.now()
```

Comment:
We should use `datetime.utcnow()` as icinga expects the time to be in UTC and makes sure its not affected by DST